### DISCUSSION TITLE
OPTIONS could easily return the default of a serializer

### DISCUSSION THREAD

**Discussion by john-parton** [encode/django-rest-framework#8003] [Ideas & Suggestions]
This only works if you apply this PR/patch: https://github.com/encode/django-rest-framework/pull/8002

If you override `rest_framework.metadata.SimpleMetadata` as follows, then you get the field default when you do an `OPTIONS` query.

```python
from rest_framework.fields import empty
from rest_framework.metadata import SimpleMetadata


class Metadata(SimpleMetadata):

    def get_field_info(self, field):
        field_info = super().get_field_info(field)

        if field.default is not empty:
            field_info['default'] = field.default

        return field_info
```

Like magic, for every field:

![Screenshot from 2021-05-21 13-16-05](https://user-images.githubusercontent.com/2071543/119181173-d9340f00-ba36-11eb-8f32-854858d0712c.png)

Here's my branch with the fix for missing `default` and the feature to add `default` to the `OPTIONS` request: https://github.com/john-parton/django-rest-framework/tree/feature/include-defaults-in-options

Should this feature be the default behavior? I would happily submit a PR

**Comment by SchrodingersGat** [comment]
I would really benefit from such a feature for the [InvenTree](https://github.com/inventree/inventree) project, as well as #8002
